src/rgw/driver/rados/rgw_bucketlogging.h

```diff
@@ -0,0 +1,71 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#pragma once
+
+#include <string>
+#include "common/ceph_time.h"
+#include "include/common_fwd.h"
+#include "common/async/yield_context.h"
+#include "rgw_common.h"
```

Comment:
nit: casn you please remove includes you don't use (or can use fwd declarations instead) in the header and move to the cc file